### DISCUSSION TITLE
Rotor in Hover

### DISCUSSION THREAD

**Discussion by iforever-yh** [byuflowlab/FLOWUnsteady#165] [Q&A]
Hello everyone,
I am studying “Rotor in Hover” case with a four blades rotor and have encountered some problems.
When I use Mid-high fidelity to calculate my rotor, the results are good for the first few revolutions, but then there is a sudden drop in thrust, and the force at the root of the blade is negative.
![屏幕截图 2024-04-20 231943](https://github.com/byuflowlab/FLOWUnsteady/assets/85104933/54b579de-14ec-4ccd-8928-810908dcfc04)

However, when I change the parameter `sigmafactor_vpmonvlm` the results seem better. 
![屏幕截图 2024-04-20 234732](https://github.com/byuflowlab/FLOWUnsteady/assets/85104933/466cb561-8e6c-47a5-928c-edcad749fd7d)

I notice that in High fidelity, the parameter `sigmafactor_vpmonvlm` has changed too.  
Why is it necessary to modify this parameter in High fidelity calculations? And how to determine the value of this parameter? 
Or there are other more complex reasons(not about `sigmafactor_vpmonvlm`) cause the problems I meet?

**Comment by inse0918** [comment]
Hi.

First, I have a question about why the thrust coefficient is so much high ?

Did you ran the original `rotorhover.jl` script and checked the coefficient of the thrust and torque ?

**Reply by iforever-yh** [reply]
>First, I have a question about why the thrust coefficient is so much high ?

I modified the source code. The output in the picture is force, not coefficient.
The radius of my rotor is 2.1m, so the result is so much high.

**Comment by EdoAlvarezR** [comment]
`sigmafactor_vpmonvlm` is a factor used to shrink or enlarge the size of particles when calculating the velocity induced by the VPM wake on the blade. In the high fidelity example we increase the factor since with higher wake resolution the particles are so small that their induced velocity might approach the singularity and cause unphysical numerical effects. 

My best guess looking at your mid-fidelity CT monitor is that the wake went numerically unstable and the simulation is on its way to blowing up. You can check that in the enstrophy monitor. Also, I recommend visualizing the simulation any time you encounter any unexpected behavior like this (e.g., check whether the hub vortex has reversed and is about to blow up).

**Reply by iforever-yh** [reply]
Thank you for your reply. I will further investigate my case according to your suggestions.
